test_conformance/integer_ops/test_int_basic_ops.c

```diff
@@ -319,7 +319,11 @@ cl_int perThreadDataInit(perThreadData * pThis, ExplicitType type,
                                           &(pThis->m_program[ i ]),
                                           &(pThis->m_kernel[ i ]), 1,
                                           &ptr, "test" );
-        test_error( err, "Unable to create test kernel" );
+        if(err) {
```

Comment:
It seems you've had to make the same change multiple times. Could the tests be refactored to avoid this?

Reply:
need to think about this more carefully